pkg/common/cns-lib/volume/manager.go

```diff
@@ -105,6 +113,8 @@ var (
 	// read/write on manager instance.
 	managerInstanceLock sync.Mutex
 	volumeTaskMap       = make(map[string]*createVolumeTaskDetails)
+	// Alias for CreateVolumeOperationRequestDetails function declaration.
+	createRequestDetails = cnsvolumeoperationrequest.CreateVolumeOperationRequestDetails
```

Comment:
Not related to this change but will it be better to refactor `CreateVolumeOperationRequestDetails` method to accept `OperationDetails` object instead of its member fields? The caller will pass the object.

Reply:
`CreateVolumeOperationRequestDetails` is meant to create a `VolumeOperationsRequestDetails` objects from its member fields :) 
FYI this function doesn't store those details on the API server. Callers (volume manager) use this object as an input to the `StoreRequestDetails` interface implementation, which then stores it on the API server.